Build.mak

```diff
@@ -1,8 +1,6 @@
 ifeq ($(DVER),1)
 override DFLAGS := $(filter-out -di,$(DFLAGS)) -v2 -v2=-static-arr-params -v2=-volatile
 else
-# Open source Makd uses dmd by default
-DC ?= dmd
```

Comment:
Typo in commit message "confusin" -> "confusing", maybe add that it's removed because is the makd default.